webpack/webpack.bzl

```diff
@@ -0,0 +1,131 @@
+"""Webpack bundle producing rule defintion."""
+
+load("@build_bazel_rules_nodejs//:providers.bzl", "DeclarationInfo", "ExternalNpmPackageInfo", "JSModuleInfo", "node_modules_aspect", "run_node")
+load("@build_bazel_rules_nodejs//internal/common:expand_variables.bzl", "expand_variables")
+load("@build_bazel_rules_nodejs//internal/linker:link_node_modules.bzl", "module_mappings_aspect")
+
+
+_ATTRS = {
+        "args": attr.string_list(
+            doc = """Command line arguments to pass to Webpack.
+
+These argument passed on the command line before arguments that are added by the rule.
+Run `bazel` with `--subcommands` to see what Webpack CLI command line was invoked.
+
+See the <a href="https://webpack.js.org/api/cli/">Webpack CLI docs</a> for a complete list of supported arguments.""",
+            default = [],
+        ),
+        "data": attr.label_list(
+            doc = """Other libraries for the webpack compilation""",
+            aspects = [module_mappings_aspect, node_modules_aspect],
+            allow_files = True,
+        ),
+        "output_dir": attr.bool(),
```

Comment:
we should include a test that uses this attribute

Reply:
Agreed. I will add more tests once we form the input and output attributes of the rule.